src/interface/src/app/plan/plan.component.ts

```diff
@@ -64,15 +55,15 @@ export class PlanComponent implements OnInit, OnDestroy {
           path: path === 'config' ? getPlanPath(plan.id) : undefined,
         },
       ];
-      if (scenario === undefined) {
-        return crumbs;
-      }
-      if (path === 'config' && !scenarioId) {
+
+      if (path === 'config' && !scenarioId && !scenarioName) {
         crumbs.push({ name: 'New Scenario' });
       }
-      if (scenario) {
-        crumbs.push({ name: scenario.name || '' });
+
+      if (scenarioName) {
+        crumbs.push({ name: scenarioName || '' });
```

Comment:
you are already checking for scenarioName so you dont need to `|| ''`

Reply:
That is true, updated, thank you Pablo!